src/Scrolling.md

```diff
@@ -4,14 +4,20 @@
 These registers can be accessed even during Mode 3, but modifications may not take
 effect immediately (see further below).
 
-## FF42–FF43 — SCY, SCX: Viewport Y position, X position
+## FF42–FF43 — SCY, SCX: Background viewport Y position, X position
 
-Those specify the top-left coordinates of the visible 160×144 pixel area within the
+These two registers specify the top-left coordinates of the visible 160×144 pixel area within the
 256×256 pixels BG map. Values in the range 0–255 may be used.
 
+The PPU calculates the bottom-right coordinates of the viewport with those formulas: `bottom := (SCY + 143) % 256` and `right := (SCX + 159) % 256`.
+As suggested by the modulo operations, in case the values are larger than 255 they will "wrap around" towards the top-left corner of the tilemap.
+
+### Visual example:
```

Comment:
```suggestion
Example from the homebrew game *Mindy's Hike*:

```

I think it's more useful to provide context for the screenshot (though a link the game is not necessary), and a blank line ensures we have correct markup.

I would even encourage using `<figure>` for this:
```md
<figure><figcaption>

Example from the homebrew game *Mindy's Hike*:

</figcaption>

...image...

</figure>
```